packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Added functionality for pagination in job spool files and created a key binding option to load more job spool records. [#1787](https://github.com/zowe/zowe-explorer-vscode/issues/1787)
```

Comment:
I don't think the keybinding is relevant in the ZE API CHANGELOG entry as it would only pertain to the ZE package.  Should this be more focused on any new or changed APIs in this package, seems like this could be the entry for ZE CHANGELOG.